waitress/static/js/dashboard.js

```diff
@@ -0,0 +1,91 @@
+// DOM elements
+const modalBody = document.getElementById('modal-body');
+const refreshSlackUserBtn = document.getElementById('refresh-slack-btn');
+
+
+const notyConfig = {
+    timeout: 1000,
+    closeWith: ['click'],
+    theme: 'relax'
+};
+
+const SPINNER_HTML = `<section class="d-flex justify-content-center align-items-center w-100">
+<div class="spinner-grow text-dark"></div>
+</section>`;
+
+const REFRESH_HTML = '<i class="fas fa-sync-alt"></i>';
+
+export async function refreshSlackUser() {
+    try {
+        refreshSlackUserBtn.innerHTML = `${SPINNER_HTML} Loading...`;
+        refreshSlackUserBtn.disabled = true;
+        await axios.put('/users/update-users/');
+        sendNotification('Slack users Refreshed Successfully');
+        refreshSlackUserBtn.innerHTML = REFRESH_HTML;
+        refreshSlackUserBtn.disabled = false;
+    } catch(error) {
+        sendNotification('error', error.message);
+        console.log(error.message);
+        refreshSlackUserBtn.innerHTML = REFRESH_HTML;
+        refreshSlackUserBtn.disabled = false;
+    }
+}
+
+function sendNotification(text, type = 'success') {
+    return new Noty({ text, type, ...notyConfig }).show();
```

Comment:
Where is `Noty` coming from? another third party global library? can this file be minified?